lib/convection/model/template/resource/aws_efs_mount_target.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        # Represents an {https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-efs-mounttarget.html
+        # Amazon EFS Mount Target}
+        class EFSMountTarget < Resource
+          type 'AWS::EFS::MountTarget'
+          property :file_system_id, 'FileSystemId'
+          property :ip_address, 'IpAddress'
+          property :security_groups, 'SecurityGroups', :type => :list
```

Comment:
Should there be a "security_group" helper function, a la https://github.com/rapid7/convection/blob/c60c2e4a22db513a4774a60a7c44c3a59c7e2d2e/lib/convection/model/template/resource/aws_api_gateway_usage_plan.rb#L18 or does that get handled automagically somehow?